parselglossy/validate.py

```diff
@@ -0,0 +1,195 @@
+import re
+from typing import Any, Dict
+
+
+JsonDict = Dict[str, Any]
```

Comment:
Nitpick: can we name the type `JSONDict`?

Reply:
I took Guido's naming: https://github.com/python/typing/issues/182#issuecomment-185996450 - but no problem for me to change it.